### PR TITLE
Add child when dojo is created

### PR DESCRIPTION
**Pull request by Wardormeur**
so that founders have the right to have a social life and children, too :o)

### CODE REVIEW

**Comment by adhorrig** [comment]
Tested and it's working 👍

File: dojos.js [hunk 1]
```diff
@@ -727,6 +732,12 @@ module.exports = function (options) {
         });
       },
       function (dojo, userDojo, cb) {
+        seneca.act({role: plugin, cmd: 'add_children_parent_dojo', userId: userDojo.userId, dojoId: userDojo.dojoId}, function (err) {
+          if (err) cb(err);
```
Comment (dberesford) [inline dojos.js]: `if (err) return cb(err);` - can lead to some strange bugs otherwise! :-)

File: dojos.js [hunk 2]
```diff
@@ -1801,40 +1812,9 @@ module.exports = function (options) {
...
-                };
-
-                seneca.act({role: plugin, cmd: 'save_usersdojos', userDojo: youthUserDojo}, cb);
-              } else {
-                return done();
-              }
-            });
-          });
-        }, done);
+      seneca.act({role: plugin, cmd: 'add_children_parent_dojo', userId: userDojo.userId, dojoId: userDojo.dojoId}, function (err) {
```
Comment (dberesford) [inline dojos.js]: you can replace the whole `function (err){..}` with just `done`

File: dojos.js (hunk 1 above)
Comment (Wardormeur) [inline dojos.js]: Good catch, thanks :)

**Pull request merged**